core/src/main/java/io/grpc/internal/FailingClientStream.java

```diff
@@ -30,27 +30,31 @@
   private boolean started;
   private final Status error;
   private final RpcProgress rpcProgress;
+  private final StatsTraceContext statsTraceContext;
 
   /**
    * Creates a {@code FailingClientStream} that would fail with the given error.
    */
-  public FailingClientStream(Status error) {
-    this(error, RpcProgress.PROCESSED);
+  public FailingClientStream(Status error, StatsTraceContext statsTraceContext) {
```

Comment:
We also pass `tracers` here instead?

Reply:
Yeah, sure. This code could create a StatsTraceContext if it wanted, for convenience (with a newly-added public constructor in StatsTraceContext)